### ISSUE TITLE
Get GCP to recognize our logs

### ISSUE THREAD

**Issue by verygoodsoftwarenotvirus** [dinnerdonebetter/dinnerdonebetter#100]
![image](https://user-images.githubusercontent.com/1289344/170195630-b4661958-6e4a-4733-9543-29c635fcc088.png)

Take the above error log for instance: this is an error log, but it's being parsed as a `Default`, which makes the other features of this service (measuring frequency/distribution of errors, etc.) useless. Should fix!

**Comment by verygoodsoftwarenotvirus** [comment]
https://cloud.google.com/logging/docs/setup/go

**Comment by verygoodsoftwarenotvirus** [comment]
I've implemented this in a few PRs (with bug fixes) #129 #128 

It's capable of being used now, but when I tried it out, I got these errors:

```
textPayload: "2022/06/17 02:00:53 logging client: rpc error: code = PermissionDenied desc = The caller does not have permission"
```

which I don't feel like dealing with right now

**Comment by verygoodsoftwarenotvirus** [comment]
I haven't actually been impeded by this
